README.md

```diff
@@ -691,6 +691,22 @@ As with any mbed CLI command, `mbed update` uses the current directory as a work
 
 <span class="tips">**Tip: Synchronizing library references:** Before triggering an update, you might want to synchronize any changes that you've made to the program structure by running ``mbed sync``, which will update the necessary library references and get rid of the invalid ones.</span>
 
+## Publishing local program or library
+
+When you create new (local) source control managed program or library, all revision history will exist only locally and the repository wouldn't be associated with remote one. To publish the local repository (while preserving the revision history) please follow these steps:
```

Comment:
When you create a new (local) source-control managed program or library, its revision history exists only locally; the repository is not associated with the remote one. To publish the local repository without losing its revision history, please follow these steps: